packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboard.tsx

```diff
@@ -76,14 +76,19 @@ export const CloudPulseDashboard = (props: DashboardProperties) => {
     isLoading: isDashboardLoading,
   } = useCloudPulseDashboardByIdQuery(dashboardId);
 
+  const platformFilter =
+    dashboard?.service_type === 'dbaas' ? { platform: 'rdbms-default' } : {};
+
   const {
     data: resourceList,
     isLoading: isResourcesLoading,
   } = useResourcesQuery(
     Boolean(dashboard?.service_type),
     dashboard?.service_type,
     {},
-    {}
+    {
+      platformFilter,
+    }
```

Comment:
```suggestion
    platformFilter
```

earlier your were spreading object's values inside the curly braces but now since you are just using the object so you don't need curly braces here